src/pages/Signup/Signup.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './Signup.scss';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
```

Comment:
import 하는 순서도 지켜주세요!
보통은 모듈 - 컴포넌트 - style파일 순서로 Import합니다!
지금같은경우엔 아래처럼 되어야겠네요
```suggestion
import React, { useState } from 'react';
import { useNavigate } from 'react-router-dom';
import './Signup.scss';
```